README.md

```diff
@@ -4,8 +4,8 @@
 
 ---
 
-[![Build Status](https://jenkins.blue-oil.org/job/blueoil_main/badge/icon)](https://jenkins.blue-oil.org/job/blueoil_main/)
-[![Build Status](https://jenkins.blue-oil.org/job/blueoil_lmnet/badge/icon)](https://jenkins.blue-oil.org/job/blueoil_lmnet/)
+[![Build Status](https://jenkins.blueoil.org/job/blueoil_main/badge/icon)](https://jenkins.blueoil.org/job/blueoil_main/)
```

Comment:
Sorry. CI badge is avairable ?

Reply:
No. in now, jenkins.blueoil.org is unavailable.
before merge this PR, we need to setup https://jenkins.blueoil.org and https://docs.blueoil.org first.